[AG-4640] Fix(sidebar): fix example

sdwvit opened the pull request:
Add Vue, Angular, and React examples for tool panels with popup/drawer functionality

fixes [AG-4640](https://ag-grid.atlassian.net/browse/AG-4640)

sdwvit committed:
Add Vue, Angular, and React examples for tool panels with popup/drawer functionality

 - Add Vue3 example with popup/drawer tool panel controls and grid setup
 - Introduce Angular component implementing columns/filters tool panels
 - Create React functional component with tool panel integration
 - Update .idea/jsLibraryMappings.xml to exclude ag-grid/node_modules
 - Fix styles.css transform value for panel positioning

sdwvit committed:
Fix tool panel top position in styles.css

StephenCooper commented on documentation/ag-grid-docs/src/content/docs/side-bar/_examples/tool-panel-parent/provided/angular/app.component.ts:
```diff
@@ -0,0 +1,167 @@
+} from 'ag-grid-community';
+
+/** ENABLE AG-TEST-ID START **/
+import * as agGridCommunity from 'ag-grid-community';
```
You will need to ensure only the test code is within this block.

StephenCooper commented on documentation/ag-grid-docs/src/content/docs/side-bar/_examples/tool-panel-parent/provided/angular/app.component.ts:
```diff
@@ -0,0 +1,167 @@
+    constructor(private http: HttpClient) {}
+
+    ngAfterViewInit() {
+        this.columnsToolPanel.parent = this.popupRef?.nativeElement.querySelector('.content') as HTMLElement;
```
Might be preferable to directly access the content element as opposed to using querySelector in angular.

StephenCooper commented on documentation/ag-grid-docs/src/content/docs/side-bar/_examples/tool-panel-parent/provided/reactFunctionalTs/index.tsx:
```diff
@@ -0,0 +1,169 @@
+    const { data, loading } = useFetchJson('https://www.ag-grid.com/example-assets/olympic-winners.json');
+
+    const columnsToolPanel = useMemo(() => {
+        console.log(popupRef.current?.querySelector('.content') instanceof HTMLElement);
```
debug code?

StephenCooper reviewed (changes requested):
Some updates please.

sdwvit committed:
Clean up test ID setup and debug logs

 - Remove debug console log from React example
 - Comment out test ID configuration in Angular component
 - Update module registrations in Angular example

sdwvit committed:
Add refs for content elements in tool panels across Vue, React, and Angular examples

 - Add popupContent/drawerContent refs in Vue3 component for direct DOM access
 - Introduce popupContentRef/drawerContentRef in React functional component
 - Add ViewChild for popupContent/drawerContent in Angular component
 - Update tool panel parent references to use content refs instead of querySelector
